charts/ingress-nginx/templates/admission-webhooks/job-patch/clusterrole.yaml

```diff
@@ -20,7 +20,7 @@ rules:
     verbs:
       - get
       - update
-{{- if .Values.podSecurityPolicy.enabled }}
+{{- if and .Values.podSecurityPolicy.enabled (not .Values.global.podSecurityStandards.enforced) }}
```

Comment:
As Giant Swarm is dropping PSP in recent releases and upstream is planning on doing so, that diff will probably disappear in the near future.